SlevomatCodingStandard/Sniffs/Classes/TraitUseSpacingSniff.php

```diff
@@ -28,6 +28,9 @@ class TraitUseSpacingSniff implements Sniff
 	/** @var int */
 	public $linesCountBeforeFirstUse = 1;
 
+	/** @var int */
+	public $linesCountBeforeFirstUseWhenFirstInClass = 1;
```

Comment:
Please change the default to `null` to be compatible with actual version. If the variable is `null`, use `linesCountBeforeFirstUse`.

Reply:
Done